_data/simple-icons.json

```diff
@@ -9719,7 +9724,8 @@
         {
             "title": "Veritas",
             "hex": "B1181E",
-            "source": "https://my.veritas.com/cs/groups/partner/documents/styleguide/mdaw/mdq5/~edisp/tus3cpeapp3855186572.pdf"
+            "source": "https://www.veritas.com/",
+            "guidelines": "https://my.veritas.com/cs/groups/partner/documents/styleguide/mdaw/mdq5/~edisp/tus3cpeapp3855186572.pdf"
```

Comment:
We should try to find a page where this document is linked

Reply:
Many of these PDFs are not linked from their website. The PDF is exposed but not available through manual navigation.
The homepage linked instead usually contains the same icon as the PDF did, so it can be used as source for any future updates.

Reply:
> The homepage linked instead usually contains the same icon as the PDF did

The PDF contains a Vector version of their logo while there are only Raster format on their homepage.

Reply:
this seems to be a problem for several of these.
We need to make a rule about this: PNG header over SVG in PDF, or viceversa? Should we accept an unofficial SVG if it is similar to a SVG inside an official PDF?

Reply:
If the icon were created from a SVG file, it feels wrong to me to set the source to a page where there are no Vector version.

Honestly, I think it would be best to open a new issue for icons where the source is not representative. We could discuss those there and try to find better alternative. Keep the changes where everyone agree in this PR so we can, at least, have updated source for many of them.

Reply:
then the PDF stays. This PDF seems to be shared from a filesystem in the Veritas sever, but I don't know if it is hosted in any webpage, specially on their website.